client/src/pages/Profile.jsx

```diff
@@ -11,7 +11,7 @@ export const Profile = () => {
   
   const generateInterests = () => {
     return interests.map((interest) => {
-      return <p key={interest.causeId}>{CAUSES.filter(cause => cause.causeId == interest.causeId)[0].causeName}</p>
+      return <p key={interest.causeId}>{CAUSES.filter(cause => cause.causeId === interest.causeId)[0].causeName}</p>

```

Comment:
To safeguard against comparing a string causeId with a number causeId, while keeping the triple equal to avoid the warning:

```javascript
{
  CAUSES.filter(
    (cause) => parseInt(cause.causeId) === parseInt(interest.causeId)
  )[0].causeName
}
```

Reply:
You can switch out this snippet or add back the == in case the 2 data types ever get out of sync 🍄✨😊

Reply:
Will probably resolve this during our next PR since that one is focused on the profile page.